doc/rose-rug-advanced-tutorials-arch.html

```diff
@@ -0,0 +1,368 @@
+            echo 'list3' >> $ROSE_DATA/file_list3
+            mkdir -p $ROSE_DATA/ARCHIVING || true
+        """
+    [[archive_files_rsync]]
+    [[archive_files_scp]]
+</pre>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Example App</h3>
+
+      <p>In the suite directory create an <samp>app</samp> directory</p>
+
+      <p>In the <samp>app</samp> directory create a <samp>archive_files_rsync</samp>
+            directory</p>
+
+      <p>In the <samp>archive_files_rsync</samp> directory create a
+      <samp>rose-app.conf</samp> file and paste in the following lines:</p>
+      <pre class="prettyprint lang-cylc">
+mode=rose_arch
+
+[env]
+ARCH_TARGET=$ROSE_DATA/ARCHIVING
+
+[arch]
+command-format=rsync -t %(sources)s %(target)s
+source-prefix=$ROSE_DATAC/
+target-prefix=$ARCH_TARGET/
+update-check=mtime+size
+
+[arch:solo.file]
+source=file_solo
+
+[arch:files]
+source=file_list1 file_list3
+source-prefix=$ROSE_DATA/
+
+[arch:dir]
+source=file*
+source-prefix=$ROSE_DATA/
+
+[arch:file_zipped.tar]
+source=file_zip
+</pre>
+      <p>In the <samp>app</samp> directory create a <samp>archive_files_scp</samp>
+                  directory</p>
+
+      <p>In the <samp>archive_files_scp</samp> directory create a
+            <samp>rose-app.conf</samp> file and paste in the following lines:</p>
+                  <pre class="prettyprint lang-cylc">
+mode=rose_arch
+
+[env]
+ARCH_TARGET=$ROSE_DATA/ARCHIVING
+
+[arch]
+command-format=scp %(sources)s %(target)s
+source-prefix=$ROSE_DATAC/
+target-prefix=$ARCH_TARGET/
+update-check=mtime+size
+
+[arch:rename/]
+rename-format=%(cycle)s_%(tag)s_%(name)s
+rename-parser=^.*list(?P&lt;tag&gt;.*)$
+source=file_list?
+</pre>
+    </div>
+
+    <div class="slide">
+      <h2 id="description">Description</h2>
+
+      <p>You have now created a suite that:</p>
+
+      <ul>
+         <li><samp>make_files</samp> sets up the files and archving directory
+        for <samp>archive_files</samp> to use.</li>
+
+        <li>contains a <samp>archive_files_rsync</samp> task that "archives", 
+        rsync's files to an ARCHIVING folder in the $ROSE_DATA directory.</li>
+        <li>contains a <samp>archive_files_scp</samp> task that "archives", 
+        scp's renamed files and moves them to an ARCHIVING folder in the 
+        $ROSE_DATA directory.</li>
+                
+      </ul>
+    </div>
+
+    <div class="slide">
+      <h3 class="alwayshidden">Run the suite</h3>
+
+      <p>Save your changes and run the suite using <samp>rose suite-run</samp>
+      .</p>
+
+      <p>View the suite output using <samp>rose suite-log</samp> and inspect
+      the output of the <samp>make_files</samp>,
+       <samp>archive_files_rsync</samp> and <samp>archive_files_scp</samp>
+      tasks.</p>
+    </div>
+
+    <div class="slide">
+      <h3>Results of "archiving"</h3>
+
+      <p>Change to the $ROSE_DATA/ARCHVING directory of the suite i.e.
+      cylc-run/SUITE_ID/share/data/ARCHIVING.</p>
```

Comment:
Wrap this with `<samp>`. (It is also worth adding a `/` at the end of a directory.)